gui/wxpython/datacatalog/catalog.py

```diff
@@ -47,6 +51,8 @@ def __init__(self, parent, giface=None, id=wx.ID_ANY,
 
         Debug.msg(1, "DataCatalog.__init__()")
 
+        delay = 2000
```

Comment:
The scope of this variable is unnecessary large. Why not to define it right before you need it for the info bars. It would help identifying what delay is that.